scripts/IceGridUtil.py

```diff
@@ -243,6 +245,7 @@ def setupClientSide(self, current):
             serverProps = Server().getProps(current)
             variables = {
                 "test.dir" : self.getPath(current),
+                "test.host": current.driver.getHost(current.config.transport, current.config.ipv6),
```

Comment:
It's odd that test.host is computed from the transport.

Reply:
Yeah, it's used for `bt`